src/wizard/Step0/components/ViewPortfolio/ViewPortfolio.vue

```diff
@@ -103,11 +31,82 @@
 <script lang="ts">
 import Vue from "vue";
 import { Component } from "vue-property-decorator";
-import { Portfolios } from "types/Portfolios";
-@Component({})
+import { Portfolio } from "types/Portfolios";
+import PortfolioSummary from "./PortfolioSummary.vue";
+import { TaskOrderDetails, TaskOrderFile } from "types/Wizard";
+
+@Component({
+  components: {
+    PortfolioSummary,
+  },
+})
 export default class ViewPortfolio extends Vue {
-  get portfolios(): Portfolios {
-    return this.$store.getters.getAllPortfolios;
+  public portfolios: Portfolio[] = [];
+
+  private mapPortfolio(item: any): Portfolio {
+    const mapTaskOrder = (taskOrderItem: any): TaskOrderDetails => {
+      if (taskOrderItem) {
+        const taskOrderFile: TaskOrderFile = {
+          id: taskOrderItem.id || "-1",
+          name: taskOrderItem.name || "",
+          description: taskOrderItem.description || "",
+          created_at: "",
+          updated_at: "",
+          size: 20000,
+          status: "",
+        };
+
+        const taskOrder: TaskOrderDetails = {
+          task_order_number: taskOrderItem.task_order_number,
+          clins: taskOrderItem.clins,
+          task_order_file: taskOrderFile,
+        };
+
+        return taskOrder;
+      }
+
+      throw new Error("invalid item");
+    };
+
+    let portfolio: Portfolio = {
+      id: item.id,
+      description: item.portfolio_step ? item.portfolio_step.description : "",
+      name: item.portfolio_step ? item.portfolio_step.name : "",
+      dod_component: item.portfolio_step
+        ? item.portfolio_step.dod_components
+        : [],
+      csp_provisioning_status: item.status,
+      portfolio_managers: item.portfolio_step
+        ? item.portfolio_step.portfolio_managers
+        : [],
+      taskOrders: item.funding_step ? [mapTaskOrder(item.funding_step)] : [],
+      applications: [],
+    };
+
+    return portfolio;
+  }
+
+  public async getPortfolios(): Promise<Portfolio[]> {
+    const apiUrl =
+      "https://s63gzoj8bh.execute-api.us-gov-west-1.amazonaws.com/prod/portfolioDrafts/";
```

Comment:
Just a thought... don't know how apiUrls will be reused/referenced down the road, but could we maybe put them in some global location somewhere instead of hardcoding them into individual files? maybe this is a one-off and would never be referenced from another file... just maybe need to start thinking about these urls now that will be actually tapping into some backend goodness

Reply:
yes, we need to give some thought to properly storing these calls.